sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveTableScanSuite.scala

```diff
@@ -187,6 +189,47 @@ class HiveTableScanSuite extends HiveComparisonTest with SQLTestUtils with TestH
     }
   }
 
+  test("SPARK-32867: When explain, HiveTableRelation show limited message") {
+    withSQLConf(HiveUtils.CONVERT_METASTORE_ORC.key -> "false",
+      "hive.exec.dynamic.partition.mode" -> "nonstrict") {
+      withTable("df") {
+        spark.range(1000)
+          .select(col("id"), col("id").as("k"))
+          .write
+          .partitionBy("k")
+          .format("hive")
+          .mode("overwrite")
+          .saveAsTable("df")
+
+        val scan1 = getHiveTableScanExec("SELECT * FROM df WHERE df.k < 3")
+        assert(scan1.simpleString(100).replaceAll("#\\d+L", "") ==
+          "Scan hive default.df [id, k]," +
+            " HiveTableRelation [" +
+            "`default`.`df`," +
```

Comment:
This seems duplicated to `Scan hive default.df`

Reply:
> This seems duplicated to `Scan hive default.df`

Yea, in origin plan, also have this problem. we can't drop this in `HiveTableRelation`  since we need to show table in Optimized plan and  analyzed plan. 

How about  implement HiveTableScanExec's `simpleString` to filter  HiveTableRelation's table info?

Reply:
let's leave it, as it's a trivial problem.

Reply:
> let's leave it, as it's a trivial problem.

Ok.